app/src/main/res/layout/activity_tab_switcher.xml

```diff
@@ -19,6 +19,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:orientation="vertical"
+    android:animateLayoutChanges="true"
```

Comment:
what’s this for?

Reply:
It's just to make the transition a bit nicer when you enter the selection mode and the bottom bar is hidden.